.nvmrc

```
@@ -1 +1 @@
-16
+16.19.0
```

Quitte à monter dans la dernière version 16

```suggestion
16.20.0
```

---

J'ai rollback ce fichier à juste `16`, comme c'était avant.
Je me dis que si on veut monter de version, ça pourrait être fait avec le passage à Nuxt 3 ?